src/include/timestamp.h

```diff
@@ -16,14 +16,14 @@
 #define WT_TS_INT_STRING_SIZE (2 * 10 + 4 + 1)
```

Comment:
Does this need to be updated to account for the durable time now?

Reply:
Based on the comment, it already take the durable time into account.

Reply:
Yes. We should change the size of WT_TP_STRING_SIZE